### ISSUE TITLE
pop-up to save wallet at start might also suggest to save private key?

### ISSUE THREAD

**Issue by jabo38** [NewEconomyMovement/NemCommunityClient#450]
I'm just wondering if there should be a message to save the private key to somehow in this warning.  I know it is a more difficult warning but some people are having problems with wallets not working for them.  
![screenshot 2016-01-06 14 23 09](https://cloud.githubusercontent.com/assets/10395907/12135630/68710b58-b481-11e5-94f4-948712895739.png)

**Comment by fyahfox** [comment]
Well, the problem is that handling the priv key is a bit more advanced and if done wrong it is a security problem. So I'm not sure. We would have to think about how to describe it very well at least...

**Comment by jabo38** [comment]
That makes sense.  I guess I was just thinking I am seeing a lot more "I am having a problem with my wallet" user fails then "I am having a problem with my key"

**Comment by fyahfox** [comment]
Right, I think that is true. But still it would be dangerouse to just display them the priv key and be like "back this up please". People will then save the priv key on the desktop and feel safe^^
